tests/util/test_yaml.py

```diff
@@ -121,13 +132,18 @@ def test_include_dir_named(self, mock_walk):
     def test_include_dir_named_recursive(self, mock_walk):
         """Test include dir named yaml."""
         mock_walk.return_value = [
-            ['/tmp', ['tmp2'], ['first.yaml']],
+            ['/tmp', ['tmp2', '.ignore', 'ignore'], ['first.yaml']],
             ['/tmp/tmp2', [], ['second.yaml', 'third.yaml']],
+            ['/tmp/.ignore', [], ['ignore.yaml']],
```

Comment:
Same here, but not sure what would be a good way to do this test, since removing it from this array is also not the answer